onnxruntime/test/optimizer/qdq_transformer_test.cc

```diff
@@ -1826,5 +1827,38 @@ TEST(QDQTransformerTests, QDQ_Selector_Test) {
     ASSERT_FALSE(result.has_value());
   }
 }
+
+TEST(QDQTransformerTests, QDQ_Shared_GetSelectors_Test) {
```

Comment:
nit, can try to combine this test case with the `QDQ_Selector_Test` test case
Can be updated in a later PR